_sass/chartButtons.scss

```diff
@@ -31,7 +31,7 @@
     display: inline-block;
     &[data-balloon]:before {
       padding: 0.5em;
-      font-size: 0.8em;
+      font-size: 13px;
```

Comment:
You can use **$FONT_SIZE_S** instead.
https://github.com/binary-com/binary-style/blob/gh-pages/_sass/constants.scss